### PR TITLE
The Plugin has been renamed following the plugin conventions

### PR DESCRIPTION
**Pull request by recena**
It is a perfect moment to do this kind of changes.

### CODE REVIEW

**Comment by recena** [comment]
@reviewbybees Could you review theses changes?

**Comment by aheritier** [comment]
@recena If you change the artifactId the plugin will appear 2 times in the update center and we won't be able to upgrade it (it will requires to be uninstall + reinstalled). Is it annoying ?

**Comment by recena** [comment]
@aheritier I agree with it but the perfect moment to do this refactor is now. Isn't that so?

**Comment by recena** [comment]
@aheritier We can rename the repo and leave the current `artifactId`

**Comment by tfennelly** [comment]
@recena I agree that your new name is more consistent, but maybe not worth it for the reasons you and @aheritier have been talking about?

On the other hand, surely it can be removed from the update center?

**Comment by recena** [comment]
@tfennelly @aheritier IMO this is the right moment to do this. Later will be more difficult.

**Comment by aheritier** [comment]
Let's go :bee:

**Comment by recena** [comment]
@aheritier We need one more ;)

**Comment by woohgit** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -8,12 +8,12 @@
     <version>1.580.3</version>
   </parent>
 
-  <artifactId>cloudbees-disk-usage-simple</artifactId>
+  <artifactId>cloudbees-disk-usage-simple-plugin</artifactId>
```
Comment (jtnord) [inline pom.xml]: this is incorrect and against conventions.  the former was correct.

**Comment by jtnord** [comment]
artifactIds should not end with plugin by convention - it is inherent in the packaging type (and the groupId)

**Pull request merged**